_posts/2018-10-04-js-testing.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title:  "Javascript Projects Testing"
+date:   2018-10-04 12:00:00 -0400
+categories: software testing js javascript jest enzyme
+author: Tania Paiva
+---
+
+Should I test this? the answer is a BIG YES. Tests are your friends, and saviors sometimes (often).
```

Comment:
😀

Reply:
I tend to prefer `**big yes**` to BIG, gets the emphasis across without looking so out of place.